### PR TITLE
tests: fix unit test

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <chris@lonng.org>

### What problem does this PR solve? 

Forgot to register test suites to unit test cases in before PR.

### Check List 

Tests 

 - Unit test

Side effects

N/A

Related changes

N/A

### CODE REVIEW

**Comment by lonng** [comment]
PTAL @kennytm @amyangfei

File: http_test.go [hunk 1]
```diff
@@ -14,11 +17,35 @@ func TestHttp(t *testing.T) {
...
+	*CheckerInfo
+}
+
+var Contains Checker = &hasPrefix{
+	&CheckerInfo{Name: "Contains", Params: []string{"obtained", "expected"}},
+}
+
+func (checker *hasPrefix) Check(params []interface{}, names []string) (result bool, error string) {
+	defer func() {
+		if v := recover(); v != nil {
```
Comment (kennytm) [inline http_test.go]: 🤔 

This is really strange, how could `strings.Contains(params[0].(string), params[1].(string))` panic uncontrollably?

File: runtime_test.go [hunk 2]
```diff
@@ -88,7 +88,45 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
+	c.Assert(fps, HasKey, "runtime-test-3")
+	c.Assert(fps, HasKey, "runtime-test-4")
+	c.Assert(fps, HasKey, "runtime-test-5")
+
+	err = failpoint.Enable("runtime-test-6", "50*return(5)->1*return(true)->1*return(false)->10*return(20)")
+	c.Assert(err, IsNil)
+	// 50*return(5)
+	for i := 0; i < 50; i++ {
+		ok, val = failpoint.Eval("runtime-test-6")
+		c.Assert(ok, Equals, true)
```
Comment (kennytm) [inline runtime_test.go]:
```suggestion
		c.Assert(ok, IsTrue)
```

File: runtime_test.go [hunk 3]
```diff
@@ -88,7 +88,45 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
+	c.Assert(ok, IsTrue)
+	c.Assert(val.(bool), Equals, true)
+	// 1*return(false)
+	ok, val = failpoint.Eval("runtime-test-6")
+	c.Assert(ok, IsTrue)
+	c.Assert(val.(bool), Equals, false)
+	// 10*return(20)
+	for i := 0; i < 10; i++ {
+		ok, val = failpoint.Eval("runtime-test-6")
+		c.Assert(ok, Equals, true)
```
Comment (kennytm) [inline runtime_test.go]:
```suggestion
		c.Assert(ok, IsTrue)
```

File: http_test.go [hunk 4]
```diff
@@ -14,11 +17,35 @@ func TestHttp(t *testing.T) {
 	TestingT(t)
 }
 
-var _ = &httpSuite{}
+var _ = Suite(&httpSuite{})
 
 type httpSuite struct{}
 
-func (s httpSuite) TestServeHTTP(c *C) {
+type hasPrefix struct {
```
Comment (kennytm) [inline http_test.go]: This type name is a huge misnomer 🙂.

```suggestion
type contains struct {
```

File: http_test.go (hunk 1 above)
Comment (lonng) [reply http_test.go]: params may not be strings 🙂

Comment (kennytm) [reply http_test.go]: But you could avoid the panic with 

```go
obtained, ok1 := params[0].(string)
expected, ok2 := params[1].(string)
if !(ok1 && ok2) {
    return false, "invalid type blah blah lah"
}
```

See the implementation of `IsTrue` for detail.

Comment (lonng) [reply http_test.go]: OK

File: runtime_test.go [hunk 5]
```diff
@@ -88,7 +88,45 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
+	// 50*return(5)
+	for i := 0; i < 50; i++ {
+		ok, val = failpoint.Eval("runtime-test-6")
+		c.Assert(ok, IsTrue)
+		c.Assert(val.(int), Equals, 5)
+	}
+	// 1*return(true)
+	ok, val = failpoint.Eval("runtime-test-6")
+	c.Assert(ok, IsTrue)
+	c.Assert(val.(bool), Equals, true)
```
Comment (amyangfei) [inline runtime_test.go]:
```suggestion
	c.Assert(val.(bool), IsTrue)
```

File: runtime_test.go [hunk 6]
```diff
@@ -88,7 +88,45 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
+	// 1*return(false)
+	ok, val = failpoint.Eval("runtime-test-6")
+	c.Assert(ok, IsTrue)
+	c.Assert(val.(bool), Equals, false)
```
Comment (amyangfei) [inline runtime_test.go]:
```suggestion
	c.Assert(val.(bool), Equals, IsFalse)
```

File: runtime_test.go [hunk 7]
```diff
@@ -88,7 +88,45 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
+	}
+	ok, val = failpoint.Eval("runtime-test-6")
+	c.Assert(ok, IsFalse)
+
+	ok, val = failpoint.Eval("failpoint-env1")
+	c.Assert(ok, IsTrue)
+	c.Assert(val.(int), Equals, 10)
+	ok, val = failpoint.Eval("failpoint-env2")
+	c.Assert(ok, IsTrue)
+	c.Assert(val.(bool), Equals, true)
```
Comment (amyangfei) [inline runtime_test.go]:
```suggestion
	c.Assert(val.(bool), Equals, IsTrue)
```

**Pull request merged**